### PR TITLE
Update default stride

### PR DESCRIPTION
**Pull request by geyyer**
- Set default stride value to -1

### CODE REVIEW

File: example/66_complex_contraction_bilinear/run_complex_contraction_bilinear_example.inc [hunk 1]
```diff
@@ -127,44 +127,47 @@ int run_complex_contraction_bilinear_example(int argc, char* argv[])
...
     DeviceMem a_device_buf_img(sizeof(ADataType) * a_ms_ks_img.mDesc.GetElementSpaceSize());
     DeviceMem b_device_buf_img(sizeof(BDataType) * b_ns_ks_img.mDesc.GetElementSpaceSize());
     DeviceMem d_device_buf_img(sizeof(DDataType) * d_ms_ns_img.mDesc.GetElementSpaceSize());
-    DeviceMem e_device_buf_img(sizeof(EDataType) * e_ms_ns_device_result_img.mDesc.GetElementSpaceSize());
+    DeviceMem e_device_buf_img(sizeof(EDataType) *
+                               e_ms_ns_device_result_img.mDesc.GetElementSpaceSize());
 
     // Intermediate Value For E Real and Img
-    DeviceMem e_device_buf_re1(sizeof(EDataType) * e_ms_ns_device_result_re.mDesc.GetElementSpaceSize());
-    DeviceMem e_device_buf_img1(sizeof(EDataType) * e_ms_ns_device_result_img.mDesc.GetElementSpaceSize());
```
Comment (hsadasiv) [inline example/66_complex_contraction_bilinear/run_complex_contraction_bilinear_example.inc]: The indentation looks off in several places in this file alone. Has clang-format /pre-commit been run on this file?

**Review by hsadasiv — COMMENTED** [review]
Looks good except for the indentations/formatting problem in run_complex_contraction_bilinear_example.inc

File: example/66_complex_contraction_bilinear/run_complex_contraction_bilinear_example.inc (hunk 1 above)
Comment (geyyer) [reply example/66_complex_contraction_bilinear/run_complex_contraction_bilinear_example.inc]: I ran the script https://github.com/ROCm/composable_kernel/blob/develop/script/clang-format-overwrite.sh and it detected some format misses in this file.

Comment (geyyer) [reply example/66_complex_contraction_bilinear/run_complex_contraction_bilinear_example.inc]: Reverted this change

**Review by illsilin — APPROVED** [review]

**Pull request merged**